web-components/src/components/buttons/SaveButton.tsx

```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from 'tss-react/mui';
+import ContainedButton from 'web-components/src/components/buttons/ContainedButton';
+import { SaveIcon } from 'web-components/src/components/icons/SaveIcon';
+import { Theme } from 'web-components/src/theme/muiTheme';
+
+interface SaveButtonProps {
+  buttonText: string;
+  isDisabled?: boolean;
+  onSave?: () => void;
+}
```

Comment:
you could simply use or extend from `ButtonProps` from MUI
and just use disabled and onClick instead of isDisabled and onClick to be more consistent